advanced/Scripts/list.sh

```diff
@@ -91,7 +91,8 @@ Options:
   -q, --quiet         Make output less verbose
   -h, --help          Show this help dialog
   -l, --list          Display all your ${listname}listed domains
-  --nuke              Removes all entries in a list"
+  --nuke              Removes all entries in a list
+  --comment "text"    Add a comment to the domain. If adding multiple domains the same comment will be used for all"
```

Comment:
```suggestion
  --comment \"text\"    Add a comment to the domain. If adding multiple domains the same comment will be used for all"
```

Sorry for the incorrect suggestion.